src/components/templates/Footer/Footer.module.css

```diff
@@ -82,7 +82,7 @@
 }
 
 @media (max-width: 500px) {
-  .footer a {
+  .footer a, .footer p {
```

Comment:
very smart decision are u Dan Abramov?

Reply:
> очень умное решение ты Дэн Абрамов?
This is Elon Musk
![maxresdefault](https://user-images.githubusercontent.com/95440091/202913181-7bb8e8e3-1b93-4e26-b6f6-4d158197b99e.jpg)